components/proxy/conf/Caddyfile

```diff
@@ -12,15 +12,15 @@
 
 	# configure plugin order
 	# https://caddyserver.com/docs/caddyfile/directives#directive-order
-	order gitpod.cors_origin            before header
-	order gitpod.workspace_download     before redir
-	order gitpod.headless_log_download	before rewrite
-	order gitpod.configcat	            before rewrite
-	order gitpod.sec_websocket_key      before header
+	order gitpod.cors_origin before header
```

Comment:
I'm assuming the formatting is based on the Caddy fmt command

Reply:
correct 👍